### ISSUE TITLE
Quartzjobs cannot store data mysql

### ISSUE THREAD

**Issue by wezell** [dotCMS/core#19314]
We updated what is stored in Quartz Job tables - we need change the job_detail field to accept a `longblob`.  See the mysql tests.

https://raw.githack.com/dotCMS/test-results/master/projects/core/current/integration/mysql/reports/html/index.html

Looks like we will need a MySQL only startup task

**Comment by fabrizzio-dotCMS** [comment]
PR: https://github.com/dotCMS/core/pull/19325

**Comment by nollymar** [comment]
Note to QA: Test an upgrade on MySQL

**Comment by nollymar** [comment]
**Passed internal QA.** After running an upgrade using MySQL, I verified the field datatype in the specified tables and now it is declared as `longblob`. Besides, this is the log output of the upgrade task that contains the fix: 

```
14:51:36.105  INFO  startup.StartupTasksExecutor - Running Upgrade Tasks: Task05390MakeRoomForLongerJobDetail
14:51:36.105  INFO  runonce.Task05390MakeRoomForLongerJobDetail - Applying `ALTER TABLE qrtz_excl_job_details MODIFY job_data LONGBLOB;` 
14:51:36.105  INFO  db.DotConnect - Executing ALTER TABLE qrtz_excl_job_details MODIFY job_data LONGBLOB;
14:51:36.142  INFO  runonce.Task05390MakeRoomForLongerJobDetail - Applying `ALTER TABLE qrtz_job_details MODIFY job_data LONGBLOB;` 
14:51:36.142  INFO  db.DotConnect - Executing ALTER TABLE qrtz_job_details MODIFY job_data LONGBLOB;
14:51:36.158  INFO  runonce.Task05390MakeRoomForLongerJobDetail - Applying `ALTER TABLE qrtz_triggers MODIFY job_data LONGBLOB;` 
14:51:36.158  INFO  db.DotConnect - Executing ALTER TABLE qrtz_triggers MODIFY job_data LONGBLOB;
14:51:36.181  INFO  runonce.Task05390MakeRoomForLongerJobDetail - Applying `ALTER TABLE qrtz_excl_triggers MODIFY job_data LONGBLOB;` 
14:51:36.181  INFO  db.DotConnect - Executing ALTER TABLE qrtz_excl_triggers MODIFY job_data LONGBLOB;
14:51:36.198  INFO  startup.StartupTasksExecutor - Database upgraded to version: 5390
14:51:36.198  INFO  startup.StartupTasksExecutor - Finishing upgrade tasks.
```

**Comment by bryanboza** [comment]
Fixed, tested upgrading from 4.2.2 to 5.3.9.
